src/components/slider/slider.scss

```diff
@@ -137,6 +148,58 @@ md-slider *::after {
   border-color: md-color($md-accent);
 }
 
+.md-slider-thumb-label {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  position: absolute;
+  left: -($md-slider-thumb-label-height / 2);
+  top: $md-slider-thumb-label-top;
+  width: $md-slider-thumb-label-width;
+  height: $md-slider-thumb-label-height;
+  border-radius: max($md-slider-thumb-label-height, $md-slider-thumb-label-width);
```

Comment:
I'm not 100% sure what this line is doing.  Will these ever not be equal?  If so, how would we want this to look?